src/ripple/core/Stoppable.h

```diff
@@ -318,6 +328,7 @@ class Stoppable
     std::condition_variable m_cv;
     std::mutex              m_mut;
     bool                    m_is_stopping = false;
+    bool hasParent_ {false};
```

Comment:
nit : stick with the same formatting/init style as the line above, or change both ?

Reply:
I'll fix the formatting. I am okay having mixed naming and initialization. That has been acceptable in the past, not sure what the consensus is today. In a former life, I would have vigorously updated the file to conform to the latest standard. One benefit I like about mixed is that it documents evolutionary changes. If you have strong feelings about the uniformity I can oblige.